### PR TITLE
1d

### PR DESCRIPTION
**Pull request by 13bscsaamjad**
added oned controller for multiple points per step. use case: cosaxs time-resolved.

### CODE REVIEW

File: sardana_adlink/ctrl/AdlinkAIOneDCtrl.py [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+class AdlinkAIOneDCtrl(OneDController):
+    """
+    This class is the Sardana CounterTimer controller for the Adlink adc
+    based counters.
+
+    The only way to use this controller is to define up to 5 channels and
+    create a measurement group where the first channel is a master channel.
+    The Adlink card works in a way where acquisition for all channels is
+    started only once and in controller this is done when StartsAll()
```
Comment (HEnquist) [inline sardana_adlink/ctrl/AdlinkAIOneDCtrl.py:70]: Something here doesn't quite make sense.

File: sardana_adlink/ctrl/AdlinkAIOneDCtrl.py [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+        """
+
+        if self._synchronization == AcqSynch.HardwareTrigger:
+            attr_name = 'C0%s_MeanValues' % (self._master_channel - 2)
+            cb = ListenerDataReady(self._index_queue, log=self._log)
+            event_type = tango.EventType.DATA_READY_EVENT
+            self._id_callback = self.AIDevice.subscribe_event(attr_name,
+                                                              event_type, cb)
+
+        # AdlinkAI Tango device has two aleatory bugs:
```
Comment (HEnquist) [inline sardana_adlink/ctrl/AdlinkAIOneDCtrl.py:291]: Will this break if the bugs in the DS were fixed?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,7 @@
 # sardana-adlink
 Repository for the Sardana plugins related to AdLink
+
+### AdlinkAIOneDCtrl
```
Comment (HEnquist) [inline README.md:11]: This seems very short! And it should mention both the controllers in the repo

File: sardana_adlink/ctrl/AdlinkAIOneDCtrl.py (hunk 2 above)
Comment (13bscsaamjad) [reply sardana_adlink/ctrl/AdlinkAIOneDCtrl.py:291]: good question, we are using the DS from Alba as well, i hope when the bugs are fixed the controller is updated as well :)

**Pull request merged**